manifests/n/Novicon/Munixo/Client/5.0.65.14161/Novicon.Munixo.Client.yaml

```
@@ -0,0 +1,23 @@
+PackageIdentifier:            Novicon.Munixo.Client
+PackageVersion:               5.0.65.14161
+PackageLocale:                en-US
+Publisher:                    Novicon
+PackageName:                  munixo Client
+License:                      Copyright (C) 2022 Novicon GmbH
```

```suggestion
Copyright:                      Copyright (C) 2022 Novicon GmbH
```
A copyright is not a license; Please add the appropriate license terms